src/layouts/hass-tabs-subpage.ts

```diff
@@ -56,39 +56,40 @@ class HassTabsSubpage extends LitElement {
       );
 
       return shownTabs.map(
-        (page, index) => html`
-          <div
-            class="tab ${classMap({
-              active: index === activeTab,
-            })}"
-            @click=${this._tabTapped}
-            .path=${page.path}
-          >
-            ${this.narrow
-              ? html`
-                  <ha-icon .icon=${page.icon}></ha-icon>
-                `
-              : ""}
-            ${!this.narrow || index === activeTab
-              ? html`
-                  <span class="name"
-                    >${page.translationKey
-                      ? this.hass.localize(page.translationKey)
-                      : name}</span
-                  >
-                `
-              : ""}
-            <mwc-ripple></mwc-ripple>
-          </div>
-        `
+        (page) =>
+          html`
+            <div
+              class="tab ${classMap({
+                active: activeTab ? page.path === activeTab.path : false,
+              })}"
+              @click=${this._tabTapped}
+              .path=${page.path}
+            >
+              ${this.narrow
+                ? html`
+                    <ha-icon .icon=${page.icon}></ha-icon>
+                  `
+                : ""}
+              ${!this.narrow || (activeTab && page.path === activeTab.path)
```

Comment:
Since it's the same object, we should be able to do just this:
```suggestion
              ${!this.narrow || page === activeTab
```

Reply:
:+1: